.github/PULL_REQUEST_TEMPLATE_nolabels

```diff
@@ -0,0 +1,13 @@
+## ATTENTION - Release Note ack required!
```

Comment:
We could use some general PR boilerplate, independent of the release note, such as linking to developer workflow docs. For now, we could point to https://github.com/kubernetes/kubernetes/blob/master/CONTRIBUTING.md. Will that link be shown once we add a template?